### PR TITLE
docs(simapp): fixed inline middleware wiring comments

### PR DESCRIPTION
**Pull request by srdtrk**
## Description

This PR fixes the inline docs on;

- [x] callback middleware stack in callback's app.go
- [x] mock middleware stack in root app.go

closes: #XXXX (I'll create the issue in a bit)

### Commit Message / Changelog Entry

```bash
docs(simapp): fixed inline middleware wiring comments
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: modules/apps/callbacks/testing/simapp/app.go [hunk 1]
```diff
@@ -522,7 +522,7 @@ func NewSimApp(
 
 	// Create Transfer Stack
 	// SendPacket, since it is originating from the application to core IBC:
-	// transferKeeper.SendPacket -> fee.SendPacket -> callbacks.SendPacket -> channel.SendPacket
+	// transferKeeper.sendTransfer -> callbacks.SendPacket -> feeKeeper.SendPacket -> channel.SendPacket
```
Comment (DimitrisJim) [inline modules/apps/callbacks/testing/simapp/app.go:525]: might be best considering its the public name available

```suggestion
	// transferKeeper.Transfer -> callbacks.SendPacket -> feeKeeper.SendPacket -> channel.SendPacket
```

File: modules/apps/callbacks/testing/simapp/app.go [hunk 2]
```diff
@@ -574,14 +574,13 @@ func NewSimApp(
...
-	// mockModule.SendPacket -> fee.SendPacket -> channel.SendPacket
+	// SendPacket, mock module cannot send packets
 
 	// OnRecvPacket, message that originates from core IBC and goes down to app, the flow is the otherway
-	// channel.RecvPacket -> fee.OnRecvPacket -> mockModule.OnRecvPacket
+	// channel.RecvPacket -> callbacks.OnRecvPacket -> fee.OnRecvPacket -> mockModule.OnRecvPacket
 
 	// OnAcknowledgementPacket as this is where fee's are paid out
-	// mockModule.OnAcknowledgementPacket -> fee.OnAcknowledgementPacket -> channel.OnAcknowledgementPacket
+	// mockModule.OnAcknowledgementPacket -> callbacks.OnRecvPacket -> fee.OnAcknowledgementPacket -> channel.OnAcknowledgementPacket
```
Comment (DimitrisJim) [inline modules/apps/callbacks/testing/simapp/app.go:583]:
```suggestion
	// mockModule.OnAcknowledgementPacket -> callbacks.OnAcknowledgementPacket -> fee.OnAcknowledgementPacket -> channel.OnAcknowledgementPacket
```

**Review by DimitrisJim — APPROVED** [review]
two minor things, othen than that, looks good.

> I'll create the issue in a bit

don't think its necessary really, you could opt not to

File: modules/apps/callbacks/testing/simapp/app.go (hunk 1 above)
Comment (damiannolan) [reply modules/apps/callbacks/testing/simapp/app.go:525]: I'd be happy with keeping it as `SendPacket` to avoid any confusion

**Review by damiannolan — APPROVED** [review]
There is this code comment below in the diff which I cannot comment on because its unchanged but for the `icaHostStack` on line 560 it states:

```
// RecvPacket, message that originates from core IBC and goes down to app, the flow is:
// channel.RecvPacket -> callbacks.OnRecvPacket -> fee.OnRecvPacket -> icaHost.OnRecvPacket
```

We can remove the callbacks bit since the host submodule isn't wrapped with callbacks :)

**Pull request merged**